objects/oplayercontroller/Step_0.gml

```diff
@@ -70,12 +70,98 @@ canBlock = false;
 inAttackState = false;
 canTurnAround = true;
 
-
 // Initialize Hurtbox Values
 hurtbox.image_xscale = 15;
 hurtbox.image_yscale = 32;
 hurtboxOffset = -8;
 
+// Handles running
+if (runButton)
+{
+	running = true;
+	
+	// All of the commented code is to limit the dash button if necessaary.

```

Comment:
This need should be evaluated during a playtest prior to merge, so the decision can be made whether this code should be used or removed. Rather than keeping this in here, you can also just make a different branch that contains this code, allowing you to quickly apply the changes without cluttering up the code.